index.html

```diff
@@ -135,11 +135,12 @@ <h1>Automatic License Plate Recognition</h1>
     <div class="row">
     <div class="col-md-6">
     <h3> <span class="glyphicons glyphicon glyphicon-map-marker"></span> Where</h3> 
-    <p>Unknown since the ALPRs are mobile. Documents acquired by the author in 2015 show that there were at least 35 vehicles in the Chicago Police Department that had ALPRs mounted to them. Chicago can also use their video surveillance network to scan license plates.</p>
+    <p>Unknown since the ALPRs are mobile. In 2019 the Chicago Police Department purchased an additional <a href ="https://news.wttw.com/2019/01/08/chicago-police-adding-200-plate-reader-vehicles-help-combat-carjackings"
+> 200 </a> ALPRs, bringing the total to over 240. Each of Chicago's 25 districts has at least six ALPR-equipped police vehicles. Chicago can also use their video surveillance network to scan license plates.</p>
     <h3> <span class="glyphicon glyphicon-piggy-bank"></span> Costs</h3> 
-    <p>CPD has spent more than <a href="http://inthesetimes.com/article/17659/cpd-tightlipped-about-use-of-license-plate-scanners-despite-new-contract">$500,000</a> on ALPR technologies in recent years.</p>
+    <p>CPD has spent more than <a href="http://inthesetimes.com/article/17659/cpd-tightlipped-about-use-of-license-plate-scanners-despite-new-contract">$500,000</a> for software to manage and analyze data from ALPRs. In addition, each device costs <a href="https://chicago.cbslocal.com/2018/02/13/cpd-license-plate-readers/"> $25,000</a>, which could bring the total expenditure to over $1.1 million.</p>
     <h3> <span class="glyphicon glyphicon-eye-open"></span> Capabilities </h3>
-    <p> Chicago Police Department purchases Motorola / PAGIS automated license plate readers. These devices track the GPS location a license plate was seen and the timestamp. The scanned plates are then checked against a Hot List that consist of plates of interest. According to FOIA documents acquired by the author, there might be as many as 300,000 license plates stored in the Hot List at any one time in Chicago - meaning that approximately 1 in 10 Chicagoans has their vehicle on the Hot List.</p>
+    <p> Chicago Police Department purchases Motorola / PAGIS automated license plate readers. These devices track the GPS location a license plate was seen and the timestamp. The scanned plates are then checked against a Hot List that consist of plates of interest. According to FOIA documents acquired by the author, there might be as many as 300,000 license plates stored in the Hot List at any one time in Chicago - meaning that approximately 1 in 10 Chicagoans has their vehicle on the Hot List. Each plate's information is stored for 365 days unless it is <a href ="http://directives.chicagopolice.org/directives/"> deemed </a> 'pertinent to criminal or civil matters,' in which case it is retained until a court orders it to be removed.</p>
```

Comment:
Not part of this review but this would be a good thing to FOIA. How many are stored older than 365 days.